annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -382,6 +382,7 @@ private VariantAnnotation getVariantAnnotation(String variant)
                     // this is thrown when the annotationJSON can't be stored by mongo
                     // due to the variant annotation key being too large to index
                     LOG.info(e.getLocalizedMessage());
+                    return variantAnnotation;
```

Comment:
To me, this looks like maybe there is a logging configuration issue when the app runs on tomcat. Can you explore this? My instance on dashi-dev is running as a jar on the command line. Maybe it does not log correctly when running on a tomcat.